src/plugins/imagery/components/ImageryViewLayout.vue

```diff
@@ -170,6 +206,13 @@ export default {
 
             return this.isPaused;
         },
+        persistVisibleLayers() {
+            if (this.domainObject.configuration) {
```

Comment:
Let's check if it's creatable. This is the proxy for editability that we typically use.
``` JavaScript
// Code is untested and probably doesn't work as written :)
let isCreatable = this.openmct.types.get(this.domainObject.type).creatable;
```